transmogrifier/sources/json/aardvark.py

```diff
@@ -30,6 +30,28 @@ def get_main_titles(cls, source_record: dict) -> list[str]:
         """
         return [source_record["dct_title_s"]]
 
+    @classmethod
+    def get_source_link(
+        cls, source_base_url: str, source_record_id: str, source_record: dict[str, JSON]
+    ) -> str:
+        """
+        Class method to set the source link for the item.
+
+        May be overridden by source subclasses if needed.
+
+        Default behavior is to concatenate the source base URL + source record id.
+
+        Args:
+            source_base_url: Source base URL.
+            source_record_id: Record identifier for the source record.
+            source_record: A BeautifulSoup Tag representing a single XML record.
+                - not used by default implementation, but could be useful for subclass
+                    overrides
+        """
+        return source_base_url + cls.get_timdex_record_id(
+            "gismit", source_record_id, source_record
+        )
```

Comment:
I like this approach.  Had thought we don't need to call another method, as we know the logic there, but this was ensures it's _tied_ to that logic should it change.